src/components/product.js

```diff
@@ -195,13 +188,11 @@ export default class Product extends Component {
 
   get carouselImages() {
     return this.model.images.map((image) => {
-      return Object.assign({}, image, {
-        // isSelected: image.id === this.currentImage.id,
-        isSelected: true,
-        // use correct image size
-        // carouselSrc: image.variants[3].src,
+      return {
+        id: image.id,
+        src: image.src,
         carouselSrc: image.src,
-      });
+      };
```

Comment:
hmmm. maybe. i think i was actually planning on using resized images for `carouselSrc`, but forgot to get back to that. good catch!